scalafmt-cli/src/main/scala/org/scalafmt/cli/ScalafmtDynamicRunner.scala

```diff
@@ -79,6 +86,8 @@ object ScalafmtDynamicRunner extends ScalafmtRunner {
 
   private final class MyInstanceSession(opts: CliOptions, instance: Scalafmt)
       extends ScalafmtSession {
+    // check config first
+    locally { instance.format(opts.configPath, opts.configPath, "") }
```

Comment:
I can't understand how `locally` helps here

Reply:
in that case, i will remove. perhaps i am still getting used to being able to invoke methods in scala which are not part of another method.